modules/store/src/store.ts

```diff
@@ -19,9 +19,6 @@ export class Store<T> extends Observable<T> implements Observer<Action> {
     this.source = state$;
   }
 
-  /**
```

Comment:
Revert this change because its in a separate PR

Reply:
Doh, I was using 2 VSCode instances and was wondering what was happening.
Sorry... 😅